back/src/main/java/com/cocktailpick/back/cocktail/domain/Cocktail.java

```diff
@@ -108,4 +112,8 @@ public boolean isSour() {
 	public boolean isBitter() {
 		return flavor.isBitter();
 	}
+
+	public boolean isAbvBetween(int max, int min) {
+		return abv >= min && abv <= max;
```

Comment:
논리연산자 부분은 우선순위를 혼동하기 쉬운 부분이라 소괄호를 쳐주는게 더 좋을것 같아보이기도 합니다.
```suggestion
		return (abv >= min) && (abv <= max);
```

Reply:
좋은 의견 감사합니다.